wiki/en/Running-a-Server.md

```diff
@@ -145,7 +138,7 @@ To upgrade your Server to a newer version, simply download a new .deb and [re-in
 
```

Comment:
(new) Line 137 conflicts with (new) line 92.

Reply:
Removed the line 137. However, Upgrading is still maintenance - but can probably be removed...